tests/Feature/FlightControllerTest.php

```diff
@@ -0,0 +1,407 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Flight;
+use Database\Factories\AirlineFactory;
+use Database\Factories\CityFactory;
+use Database\Factories\FlightFactory;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use PHPUnit\Framework\Attributes\DataProvider;
+use Tests\TestCase;
+
+class FlightControllerTest extends TestCase
+{
+    use RefreshDatabase;
+
+    private $baseUrl = '/api/flights';
```

Comment:
```suggestion
    private const BASE_URL = '/api/flights';
```
For things like this, that we know are values that will not change, use constants. To access a constant you have to either do `self::BASE_URL` or `static::BASE_URL`. You can read about what the difference is [here](https://masteringbackend.com/posts/self-static-and-parent-in-php).